### PR TITLE
refactor(pkg): Prune transferred pkgs

### PR DESCRIPTION
**Pull request by simar7**
As part of https://github.com/aquasecurity/defsec/pull/1499 we've moved back some logic into defsec as it is shared.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,23 +6,21 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/Masterminds/semver v1.5.0
 	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/aquasecurity/defsec v0.93.2-0.20231208181342-318642ac6f08
+	github.com/aquasecurity/defsec v0.93.2-0.20240111013720-960ecc7915ab
```
Comment (simar7) [inline go.mod:9]: we'll also need to merge [that](https://github.com/aquasecurity/defsec/pull/1516) PR before merging this one.

File: .github/workflows/verify-schema.yml [hunk 2]
```diff
@@ -1,19 +0,0 @@
-name: verify schema
```
Comment (nikpivkin) [inline .github/workflows/verify-schema.yml:1]: Why is this workflow being deleted?

Comment (simar7) [reply .github/workflows/verify-schema.yml:1]: Haven't we moved the schema back to defsec? https://github.com/aquasecurity/defsec/pull/1517

Correct me if I'm wrong, as this PR removes the schema workflow and the schema itself from trivy-iac.

**Comment by simar7** [comment]
Closing in favor of https://github.com/aquasecurity/trivy-iac/pull/80

**Pull request closed without merging**